saleor/order/events.py

```diff
@@ -32,26 +32,84 @@ def _user_is_valid(user: UserType) -> bool:
     return bool(user and not user.is_anonymous)
 
 
-def email_sent_event(
-    *,
-    order: Order,
-    user: Optional[UserType],
-    email_type: str,  # use "OrderEventsEmails" class
-    user_pk: int = None,
-) -> OrderEvent:
+def notification_order_refund_event(
```

Comment:
As we spoke with Maciej, it would be nice to unify the naming of these event functions to start with the `event_` prefix so that it's easier to understand these functions when they are used in different places in the codebase. This could be renamed to: `event_order_refunded_notification`.